ios/scaffold/apple-app-site-association

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can we add a reference to this in the README of the Scaffold?  Even if it's just a pointer to our Astro docs that deal with deep linking just so somebody generating an app has a place to start from (too bad we can't put comments in this file... but alas... JSON).